src/job/fm-file-ops-job-xfer.c

```diff
@@ -866,7 +871,7 @@ gboolean _fm_file_ops_job_move_run(FmFileOpsJob* job)
                                               -1, NULL, NULL, NULL);
             /* gvfs escapes it itself */
         else /* move from virtual to native/virtual */
-            tmp_basename = fm_uri_subpath_to_native_subpath(fm_path_get_basename(path), NULL);
+            tmp_basename = fm_uri_subpath_to_native_subpath(fm_path_display_basename(path), NULL);
```

Comment:
Good attempt but this will create a memory leak each time this line is executed as fm_path_display_basename() allocates memory which will be never freed.
Thank you.